### PR TITLE
For#3278 - Updates strings in preferences

### PR DESCRIPTION
**Pull request by boek**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
    - string updates
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - #3865 - Added a new nightly icon and app name
 
 ### Changed
+- #3278 - Updates strings in preferences
```
Comment (colintheshots) [inline CHANGELOG.md]: We really need to restructure our Changelog. It's an arbitrarily-ordered list right now that's very hard to read. We should just start over with a date-based list or one entry per major release.

Comment (boek) [reply CHANGELOG.md]: @colintheshots ahah, I had the same thought when I added this line.

**Comment by boek** [comment]
@colintheshots good point! Yes, good timing with the changes to how we're going to use milestones going forward

**Comment by boek** [comment]
Follow up issue here: https://github.com/mozilla-mobile/fenix/issues/4185

**Pull request merged**